Classes/Views/UIColor+Hex.swift

```diff
@@ -18,7 +18,7 @@ extension UIColor {
             cString.remove(at: cString.startIndex)
         }
 
-        if ((cString.characters.count) != 6) {
+        if ((cString.count) != 6) {
```

Comment:
`if cString.count != 6 {`?

Reply:
Can clean up all of them here I suppose. Think Ryan coped this straight from SO 😜

Reply:
Noo Ryan wouldn't do that... would he?? 😉

Reply:
So: omit `:String` in line 15
Omit `()` in 17
Omit `(())` in 21
Omit `UIColor` in 22.